tests/unit/components/infinity-loader-test.js

```diff
@@ -191,3 +162,76 @@ test('it checks if in view after model is pushed', function(assert) {
     setTimeout(pushModelAsynchronously);
   }
 });
+
+
+test('hideOnInfinity => true : it will hide itself when inifinity is reached', function(assert) {
+  assert.expect(2);
+
+  var infinityModelStub = [
+    {id: 1, name: 'Tomato'},
+    {id: 2, name: 'Potato'}
+  ];
+
+  var component = this.subject({
+    infinityModel: infinityModelStub,
+    hideOnInfinity: true,
+  });
+
+  this.render();
+
+  assert.ok(component.get('isVisible'));
+
+  Ember.run(function() {
+    component.set('infinityModel.reachedInfinity', true);
+  });
+
+  assert.notOk(component.get('isVisible'));
+});
+
+test('hideOnInfinity => false : it will not hide itself when inifinity is reached', function(assert) {
```

Comment:
What do you think about combining both the false && default tests into one?

Reply:
No prob. And... Done.